docker/runtime/python-2.7/kubeless.py

```diff
@@ -1,31 +1,24 @@
 #!/usr/bin/env python
 
-import sys
 import os
 import imp
 
-from bottle import route, run, request
+import bottle
```

Comment:
ok, I used to do the `import` but I was convinced the other way around with the argument that only the required functions would get loaded.